src/libraries/Common/tests/TestUtilities/System/PlatformDetection.Unix.cs

```diff
@@ -241,15 +241,22 @@ private static DistroInfo GetDistroInfo()
             }
             else if (File.Exists("/etc/os-release"))
             {
-                foreach (string line in File.ReadAllLines("/etc/os-release"))
+                foreach (ReadOnlySpan<char> line in File.ReadAllLines("/etc/os-release"))
                 {
                     if (line.StartsWith("ID=", StringComparison.Ordinal))
                     {
-                        result.Id = line.Substring(3).Trim('"', '\'');
+                        result.Id = line[3..].Trim(['"', '\'']).ToString();
```

Comment:
This needs code needs to target netstandard2.0. It cannot use .NET Core specific APis and language features.

Reply:
(This is test code. We do not care about it being high perf.)

Reply:
Ah! Ok. I'll do the simple/naive thing. I did run the build but didn't get that feedback.